features/embed-video.md

```diff
@@ -0,0 +1,72 @@
+# Embed video
+
+Embedding a video from ImageKit allows you to integrate the video directly onto your platform without requiring an additional video player. Typically, the code to embed a video appears as:
+
+{% code overflow="wrap" %}
+```
+<iframe width="560" height="315" src="https://imagekit.io/player/embed/demo/sample-video.mp4?thumbnail=https%3A%2F%2Fik.imagekit.io%2Fdemo%2Fsample-video.mp4%2Fik-thumbnail.jpg" title="ImageKit video player" frameBorder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen"> </iframe>
+```
+{% endcode %}
+
+[Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) can be enabled for embedding videos using either either HTTP Live Streaming (HLS) or Dynamic Adaptive Streaming over HTTP (MPEG-DASH) protocol.
+
+### How do you generate embed video code?
+
+Embed video code can be generated for any video from the media library dashboard and from the files detail page.
+
+{% hint style="info" %}
+The embed feature is exclusive for published video files
+{% endhint %}
+
+#### Media library dashboard
+
+To generate the embed video code from the media library dashboard, hover over the lower right section of the video or right-click on the video file and select the "Embed" option.\
```

Comment:
Why is there an extra `\` at the end?

Reply:
got added while editing through gitbook , I will remove it

Reply:
done